src/misc/layout.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Layout = (props) => {
+const layout = (props) => {
```

Comment:
for convention this should be an uppercase thing, because at the end of the day it is a component